cdc/metrics/alertmanager/ticdc.rules.yml

```diff
@@ -2,111 +2,99 @@ groups:
 - name: alert.rules
   rules:
   - alert: cdc_multiple_owners
-    expr: sum(rate(ticdc_owner_ownership_counter[30s])) >= 2
```

Comment:
Suggest:
1. Rename the file to `tikv-cdc.rules.yml`.
2. Change the alert names to avoid conflict with TiCDC.
3. Change "cdc" to "TiKV-CDC" in `summary` to avoid misunstanding of user.

Reply:
Good suggest, modified, thanks!